packages/tab-bar/src/mwc-tab-bar-base.ts

```diff
@@ -134,6 +134,7 @@ export class TabBarBase extends BaseElement {
       },
       getFocusedTabIndex: () => {
         const tabElements = this._getTabs();
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
         const activeElement = (this as any).getRootNode().activeElement;
```

Comment:
This should be `(this.getRootNode() as ShadowRoot).activeElement`, and then the suppression shouldn't be needed.

Reply:
About this:

>Property 'getRootNode' does not exist on type 'TabBarBase'.

<img width="745" alt="Screenshot 2019-08-15 at 18 42 17" src="https://user-images.githubusercontent.com/1007051/63110768-758cd000-bf8c-11e9-8181-b58ff7e54dd5.png">

Reply:
Works for me :) I think you maybe need to fix some of the other `tsc` errors first. Also note I needed to make it `const activeElement = (this.getRootNode() as ShadowRoot).activeElement as Tab;` to make the next line happy.